ObjectsAsAPI/Models/CreateOrder.cs

```diff
@@ -0,0 +1,54 @@
+﻿using MongoDB.Bson;
+using ObjectsAsAPI.Services;
+using Realms;
+
+namespace ObjectsAsAPI.Models;
+
+public partial class CreateOrderPayload : IRealmObject, IPayload
+{
+    [PrimaryKey]
+    [MapTo("_id")]
+    public ObjectId Id { get; set; } = ObjectId.GenerateNewId();
+
+    [MapTo("_creatorId")]
+    public string CreatorId { get; set; }
+
+    [MapTo("content")]
+    public OrderContent? Content { get; set; }
+
+    public CreateOrderPayload()
+    {
+        if (RealmService.CurrentUser == null)
+        {
+            throw new Exception("Login before using models!");
+        }
+
+        CreatorId = RealmService.CurrentUser.Id;
+    }
+}
+
+public partial class CreateOrderResponse : IRealmObject, IResponse
+{
+    [PrimaryKey]
+    [MapTo("_id")]
+    public ObjectId Id { get; private set; }
+
+    [MapTo("_creatorId")]
+    public string CreatorId { get; private set; } = null!;
+
+    [MapTo("order")]
+    public Order? Order { get; private set; }
+
+    [MapTo("status")]
+    private string _Status { get; set; } = null!;
+
+    public ResponseStatus Status
+    {
+        get => string.IsNullOrEmpty(_Status) ? ResponseStatus.Approved : Enum.Parse<ResponseStatus>(_Status);
```

Comment:
This here has been done like to avoid the situation in which we get a response, and the properties are not set yet. If `_Status` is empty, then we'll get an exception here.
Responses are created on the backend in a trigger, and I suppose that the "create response" and "fill response" changes can be split into different changesets.